test/lib/code_corps/emails/base_email_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CodeCorps.Emails.BaseEmailTest do
+  use CodeCorps.ModelCase
+  use Bamboo.Test
+  alias CodeCorps.Emails.BaseEmail
+
+  test "get_name returns there on nil name" do
+    user = %CodeCorps.User{}
+    assert BaseEmail.get_name(user) == "there"
+  end
+
+  test "get_name returns first_name of user" do
+    f_name = "Zacck"
+    user = %CodeCorps.User{first_name: f_name}
```

Comment:
You don’t need the intermediate variable here. Can just put the string in directly as the value.

Reply:
Ignore this. Refer to my example instead.